File: ios/BaseiOSUIKit/BaseiOSUIKit/Core/Domain/Model/CoinDetailModel.swift
//
//  CoinDetailModel.swift
//  BaseiOSUIKit
//
//  Created by Ali Taha on 5.12.2023.
//

import Foundation

struct CoinDetailModel: Identifiable {
    let id: String
    var name: String = Constants.EMPTY_STRING
    var symbol: String = Constants.EMPTY_STRING
    var message: String = Constants.EMPTY_STRING
    var description: String = Constants.EMPTY_STRING
    var isActive: Bool = false
    var type: CoinType = .invalid
}

File: android/BaseAndroidComposeKtor/app/src/main/java/dev/subfly/baseandroidcomposektor/core/domain/model/CoinDetailModel.kt
package dev.subfly.baseandroidcomposektor.core.domain.model

import dev.subfly.baseandroidcomposektor.core.util.CoinType
import dev.subfly.baseandroidcomposektor.core.util.Constants

data class CoinDetailModel(
    val id: String,
    val name: String = Constants.EMPTY_STRING,
    val symbol: String = Constants.EMPTY_STRING,
    val message: String = Constants.EMPTY_STRING,
    val description: String = Constants.EMPTY_STRING,
    val isActive: Boolean = false,
    val type: CoinType = CoinType.INVALID,
)
